exonum/src/runtime/mod.rs

```diff
@@ -186,6 +186,10 @@ pub trait Runtime: Send + Debug + 'static {
     fn start_service(&mut self, spec: &InstanceSpec) -> Result<(), ExecutionError>;
 
     /// Configure a service instance with the given parameters.
+    /// 
+    /// The configuration parameters passed to the method are not saved anywhere, 
```

Comment:
```suggestion
    /// The configuration parameters which passed to the method are not saved anywhere, 
```

Reply:
Not sure that it is valid, it must be `parameters which are passed`. The current variant `parameters passed` is also valid though, iirc. @Maria-Nosyk ?

Reply:
From the grammar point of view @vitvakatu  is tight. However, it is recommended to avoid negative verb forms, epecially such a large ones. In the present case I would suggest something like 
```suggestion
    /// The configuration parameters passed to the method are deleted immediately. 
```

Reply:
"deleted" is not proper verb in this context, I suggest to use "forgot", "discarded" or something like that.

Reply:
"discarded" is a nice one!